### PR TITLE
Fix import success

### PR DESCRIPTION
**Pull request by anthony-sarkis**
_This builds on #777 so probably best to review that first_

### Automatic status checking
Now, it will automatically check if Walrus is up before the user even starts to work. 
This is mainly in the context of first time installs.

automatic service down detection

### Declare Success is better
Before it was assuming success if not actively uploading. In this V2 iteration we improve it to seperate the success concern from the progress concern.
declare success event

### Expressly render errors 
In fetch data and dropzone cases
Example, walrus is killed halfway through upload
check connection to walrus
dropzone case

### CODE REVIEW

**Comment by anthony-sarkis** [comment]
closes #775 

@PJEstrada  You know more about this section of the system so please check if it covered the cases / didn't break anything please :)

File: frontend/src/components/input/new_or_update_upload_screen.vue [hunk 1]
```diff
@@ -144,7 +144,9 @@
                               @vdropzone-removed-file="file_removed"
                               v-on:vdropzone-thumbnail="thumbnail"
                               @vdropzone-sending="drop_zone_sending_event"
-                              @vdropzone-complete="drop_zone_complete">
+                              @vdropzone-complete="drop_zone_complete"
+                              @vdropzone-success="drop_zone_success"
+                              @vdropzone-error="drop_zone_error">
```
Comment (PJEstrada) [inline frontend/src/components/input/new_or_update_upload_screen.vue:149]: Nice! I did not know about this event handler

File: install.py [hunk 2]
```diff
@@ -377,22 +377,22 @@ def populate_env(self):
             env_file = f"GCP_SERVICE_ACCOUNT_FILE_PATH={self.gcp_credentials_path}\n"
             env_file += f"CLOUD_STORAGE_BUCKET={self.bucket_name}\n"
             env_file += f"ML__CLOUD_STORAGE_BUCKET={self.bucket_name}\n"
-            env_file += 'SAME_HOST=False\n'.format(self.bucket_name)
```
Comment (PJEstrada) [inline install.py:380]: thanks for fixing this

**Review by PJEstrada — APPROVED** [review]
Thanks Anthony! Everything looks good to me, that dropzone handler is going to be super usefult for catching uploads in progress.

**Pull request merged**